client/cache/bloom_filter_cache.go

```diff
@@ -0,0 +1,84 @@
+package cache
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/beego/beego/v2/core/berror"
+	"github.com/bits-and-blooms/bloom/v3"
+)
+
+type BloomFilterCache struct {
+	Cache
+	*bloom.BloomFilter
+	loadFunc       func(ctx context.Context, key string) (any, error)
+	expiration     time.Duration // set cache expiration, default never expire
```

Comment:
I think users must pass the expiration when they call the NewBloomFilterCache because we don't know how to handle the expiration. Using default value looks not elegant